src/Pagination/PaginationManipulator.php

```diff
@@ -103,17 +103,25 @@ protected function registerConnection(
         $connectionFieldName = addslashes(ConnectionField::class);
 
         $connectionType = PartialParser::objectTypeDefinition("
```

Comment:
We can use HEREDOC syntax here to avoid having to escape the double quotes but still be able to interpolate variables.

Reply:
Done!